### PR TITLE
Wiremock structure

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: scanner/src/test/kotlin/storages/ClearlyDefinedStorageTest.kt [hunk 1]
```diff
@@ -225,15 +228,15 @@ private fun assertCurrentTime(time: Instant) {
...
+    val templateFile = File("$TEST_FILES_ROOT/cd_definitions.json")
     return templateFile.readText()
 }
 
 class ClearlyDefinedStorageTest : WordSpec({
     val wiremock = WireMockServer(
         WireMockConfiguration.options()
             .dynamicPort()
-            .usingFilesUnderDirectory("src/test/assets/")
+            .usingFilesUnderDirectory(TEST_FILES_ROOT)
```
Comment (oheger-bosch) [inline scanner/src/test/kotlin/storages/ClearlyDefinedStorageTest.kt:239]: Is this configuration option needed here? It does not seem to be defined for the other tests.

**Review by oheger-bosch — APPROVED** [review]
Changes look good to me.

There is still some redundancy in the WireMock tests, e.g. to declare the server and start and stop it. Would it be possible (in a later PR) to extract this into a helper class or a new test base class?

File: scanner/src/test/kotlin/storages/ClearlyDefinedStorageTest.kt (hunk 1 above)
Comment (sschuberth) [reply scanner/src/test/kotlin/storages/ClearlyDefinedStorageTest.kt:239]: I believe it's needed (as we use "assets", not "resources"), and I also see `usingFilesUnderDirectory` being called in other tests like `VulnerableCodeTest`, `JiraNotifierTest` etc.

**Comment by sschuberth** [comment]
> Would it be possible (in a later PR) to extract this into a helper class or a new test base class?

Probably. I haven't looked at that, but just did a search-and-replace, more or less. Any improvements you see fit are welcome in the form of a PR 😉

**Pull request merged**